plugin.py

```diff
@@ -75,6 +78,8 @@ def name(self) -> str:
     def config(self) -> ClientConfig:
         settings = sublime.load_settings("LSP-eslint.sublime-settings")
         client_configuration = settings.get('client')
+        if client_configuration is None:
+            client_configuration = {}
```

Comment:
This fixes a possible `None` type at line 125.

Reply:
Does `settings.get` accepts an optional second argument for the default value  `settings.get('client', {})`? :-)

Reply:
Yes, but mypy still complains if I do that :(

Reply:
If I find a way to properly type hint settings.get, I will let you know :-)

Reply:
Typescript has a feature called Conditional Types. You can look it up on the net if you are interested. 🙂

I was looking if mypy had something similar, but I couldn't find anything. That would allow for that.

Reply:
Maybe something like this?
```
from typing import TypeVar

T = TypeVar('T')

class Settings:
    def get(self, key: str, default: T = ...) -> T:
        ...
```

Reply:
We define a generic type T.

So when we call settings.get 
and pass a `default` value, for example 
`line_height = settings.get('line_height', 22)`
`line_height` should infer the type of the `default` value, 22 in this case, theoretically

Reply:
> line_height should have a type of int :)

Nope. What if I had JSON like this:
```
{
  "settings":
  {
    "line_height": "foo"
  }
}
```
then `settings.get('line_height', 22)` would return a `str`, not an `int`.

Reply:
So really, checking for `None` is not sufficient, but at least it stops mypy from complaining